optimum/habana/sentence_transformers/st_gaudi_trainer.py

```diff
@@ -317,9 +317,10 @@ def compute_loss(
         # Hackishly insert the distributed model into the loss function, if the loss stores the model
         # Only called once per process
```

Comment:
```suggestion
        # Insert the wrapped (e.g. distributed or compiled) model into the loss function,
        # if the loss stores the model. Only called once per process
```

Please correct the comment

Reply:
Also put the comment in `compute_loss` doc from which version you adapted these changes. My assumption is https://github.com/UKPLab/sentence-transformers/blob/845dd548f4f04ae0538fda656438298dc189ece3/sentence_transformers/trainer.py#L305

Reply:
https://github.com/UKPLab/sentence-transformers/blob/v3.1.0/sentence_transformers/trainer.py

Reply:
Yes it is from the source you listed, they tagged recently, Added tagged link to comment.